src/test.rs

```diff
@@ -14,9 +14,9 @@ use std::vec::Vec;
 #[test]
 fn it_fails_decode_with_invalid_constraints() {
     let encoded_cennznut: Vec<u8> = vec![
-        0, 0, 0, 64, 109, 111, 100, 117, 108, 101, 95, 116, 101, 115, 116, 0, 0, 0, 0, 0, 0, 0, 0,
-        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 64, 109, 101, 116, 104, 111, 100, 95, 116, 101, 115,
-        116, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0, 0x80, 64, 109, 111, 100, 117, 108, 101, 95, 116, 101, 115, 116, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
Would prefer if we didn't mix hex values in here

Reply:
Done